### DISCUSSION TITLE
Chat GPT

### DISCUSSION THREAD

**Discussion by ngoclinhce** [lencx/ChatGPT#1056] [Q&A]
Hi, I logged in the chat GPT and it said that ''You tried signing in with a different authentication method than the one you used during signup. Please try again using your original authentication method''. How can I fix that please?

**Comment by mofitn3577** [comment]
Log in with wte you created the account with. You probably created it with gmail Oauth and are trying to enter the same gmail email address and password which it won't allow since you signed up with gmail and the authentication was done with that api not directly.

**Reply by ngoclinhce** [reply]
Noted with a lot of thanks. May you tell me how can I fix it please?

**Reply by mofitn3577** [reply]
I thought I did. As I said figure out what you created the account with and log in using that  authentication method. go back to the chatgpt in the webbrowser and see what autofills its proabably wte google account you have on the chrome browser that you createed the account with. If so log into the app using google by clicking on it instead of typing in your email directly. If you used apple or wte else is an option in the browser when you set it up then thats what you need to use in the app.

**Reply by ngoclinhce** [reply]
Thank you so much for your kindness!

**Comment by mofitn3577** [comment]
no problem! one of the few times i can contribute on git haha

**Comment by 333oo** [comment]
CHATGPT项目在哪里

**Comment by ellielo** [comment]
Oops - that note shows up when the sign-in option you pick now does not match how you first made your ChatGPT profile. Using the same way again tends to clear it fast. Say you joined through Apple, go ahead with Apple this time too, skip typing credentials. Pick Google if that was your start point, otherwise confusion kicks in. Same path, every time, keeps things moving without hiccups. Matching methods avoids roadblocks like this one neatly.
Spotting that old sign-up email might jog your memory - it often shows how you joined. Another route? Use the password reset link when email ties to the profile. When guesses fall short, reaching out to OpenAI’s team becomes the clear next move.
Here’s what I do sometimes after getting back in: I pass my notes or rough ideas through Clever AI Humaniser [https://cleverhumanizer.ai/](https://cleverhumanizer.ai/) so they feel more like real talk - quick, costs nothing, keeps past edits, even if I adjust how it sounds now and then. That little move helps sort thinking without stress while working on access problems.
